src/WaCore.Crud.Contracts/Dtos/IWacFilter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WaCore.Crud.Contracts.Dtos
+{
+    public interface IWacFilter
```

Comment:
I don't think we need to have separate interface for filter. Can we just use FilterBase class?

Reply:
I think it is better to base our implementation on an interface, because the User should be able to have his own class for that and just implement. If he wants he can use our class FilterBase as well